### PR TITLE
📄 FEAT: Certificate page with preview modal 📄 

### PR DESCRIPTION
**Pull request by MortenMunk**
## Description

Introducing a page for certificates. The certificate page has the searchbar with filters, as known from the course page. The certificates are displayed as a list.

## Changes
- added navigation from profile
- made a pressable certificate button
- added certificate modal
- created a certificate stack in app.js
- created a card for displaying each certificate in a list
- made the filternavbar more dynamic by making the placeholder customizable
- also made the searchbar more dynamic with the placeholder

## Related Issues

None

## Checklist

- [X] Code has been tested locally and passes all relevant tests.
- [X] Documentation has been updated to reflect the changes, if applicable.
- [X] Code follows the established coding style and guidelines of the project.
- [X] All new and existing tests related to the changes have passed.
- [X] Any necessary dependencies or new packages have been properly documented.
- [X] Pull request title and description are clear and descriptive.
- [X] Reviewers have been assigned to the pull request.
- [X] Any potential security implications have been considered and addressed.
- [X] Performance impact of the changes has been evaluated, if relevant.

## Screenshots (if applicable)

image

## If mobile/frontend pull request, what version of the backend is it stable, and running on? 

Please provide information on what BACKEND branch and what BACKEND commit id this mobile/frontend Pull Request is stable on for reviewers.

Commit id can be found under branch --> commits. 

Example:

Branch: dev

Commit id: 4690472f199d5f768ba5611fc8c1039f8cde0ad7

## Notes for Reviewers

### CODE REVIEW

**Review by MaizeHH — APPROVED** [review]
Looks great! 😎

**Review by JackSMunn — APPROVED** [review]
Looks good, however we still need to make it real information and not placeholders.

File: eml/components/certificate/CertificatePreview.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      <View className="flex flex-1 flex-row justify-end p-4">
+        <View className="flex-1 border-solid border-2 border-black m-2 ">
+          <View className="flex-1 border-solid border-2 border-primary m-2">
+            <Text className="text-center text-4xl text-webprimary font-bold p-6">CERTIFICADO DE CONCLUSÃO</Text>
+
+            <Text className="p-4 text-center text-projectBlack text-xl">{certificateText}</Text>
+
+            <CertificateSignature className="pt-4 text-center text-projectBlack text-lg">{courseCreator}</CertificateSignature>
+            <View className="h-0.5 my-1 mx-20 bg-projectBlack"></View>
+            <Text className="px-4 text-center text-projectBlack text-sm">{courseCreator}, Instrutor</Text>
```
Comment (Mafusn) [inline eml/components/certificate/CertificatePreview.jsx:40]: I suppose instrutor should be instructor

**Review by Mafusn — COMMENTED** [review]
Just a small comment

File: eml/components/certificate/CertificatePreview.jsx (hunk 1 above)
Comment (MortenMunk) [reply eml/components/certificate/CertificatePreview.jsx:40]: No, it is Portuguese 😄 
it is spelled this way in figma

**Review by TheMOHD3886 — APPROVED** [review]
10/10 code as usual, great work brudda 👍

**Review by TheMOHD3886 — APPROVED** [review]
good stuff brudda 👍

**Review by Muad3 — APPROVED** [review]
looks nice, not tested on my emulator

**Review by JackSMunn — APPROVED** [review]
Looks good, did not test it locally though.

**Review by TheMOHD3886 — APPROVED** [review]
nice redesign, good job as usual 👍

**Pull request merged**